tests/v2/validation/registries/registry_test.go

```diff
@@ -39,6 +29,9 @@ const (
 	corralEcr              = "corralecr"
 	systemRegistry         = "system-default-registry"
 	namespace              = "fleet-default"
+	rke1ClusterType        = "rke1"
+	rke2ClusterType        = "rke2"
+	k3sClusterType         = "k3s"
```

Comment:
@izaac please review to make sure. Also since these cluster types are used across several packages we could have these as immutable strings. These consts might already exists. Can you check?

Reply:
updating to pull these names from crud package in validations/extensions